shared/bundle_analysis/comparison.py

```diff
@@ -40,21 +40,23 @@ class BundleAnalysisComparison:
     Compares bundle reports on two different commits.
     """
 
-    def __init__(self, loader: BundleReportLoader, base_sha: str, head_sha: str):
+    def __init__(
+        self, loader: BundleReportLoader, base_report_key: str, head_report_key: str
```

Comment:
You mention
> We could technically have multiple bundle reports per commit (via report_code)

The default `report_code` is `None`. In this case the `base_report_key` will be `"None"` or you'll be passing the commit sha to be the default key?

I suppose it's something the processing code will take core of, and it doesn't really matter. I'm just curious 👀

Reply:
The report key could be something like `commit_report.external_id` which seems to always be a UUID

Reply:
I like that... cool cool